src/components/Onfido/BaseOnfidoWeb.js

```diff
@@ -86,18 +85,15 @@ function initializeOnfido({sdkToken, onSuccess, onError, onUserExit, preferredLo
             onSuccess(data);
         },
         onError: (error) => {
-            const errorMessage = lodashGet(error, 'message', CONST.ERROR.UNKNOWN_ERROR);
             const errorType = lodashGet(error, 'type');
+            if (errorType === CONST.WALLET.ERROR.ONFIDO_USER_CONSENT_DENIED) {
+                onUserExit();
+                return;
+            }
+            const errorMessage = lodashGet(error, 'message', CONST.ERROR.UNKNOWN_ERROR);
             Log.hmmm('Onfido error', {errorType, errorMessage});
```

Comment:
Can we move this line above? So we can log something even if the error is `ONFIDO_USER_CONSENT_DENIED`?

Reply:
Updating